src/components/Authentication/Login.vue

```diff
@@ -27,6 +27,9 @@
         <div class="text-center">
           <button type="submit" class="btn btn-primary falko-button" id="loginButton" >Log In</button>
         </div>
+        <div class="text-center">
+          <router-link to="/users/forgotpassword"> Forgot your password?</router-link>
```

Comment:
You should avoid this hardcoded routes, use the target component instead. If needed, [here's some documentation](https://router.vuejs.org/api/#router-link)

Reply:
Yeah, i understood why. Thanks for pointing that out @MatheusRich.